modules/CameraCapture/CameraCapture.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file   AreaACameraAction.cpp
+ * @brief  カメラを制御するクラス
+ * @author Hara1274
+ */
+
+#ifndef CAMERACAPTURE_H
+#define CAMERACAPTURE_H
+
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include <vector>
+#include <string>
+#include <thread>
+#include <chrono>
+
+class CameraCapture {
+ private:
+  cv::VideoCapture cap;
+  int cameraID;
+
+ public:
+  CameraCapture();
+
+  /**
+   * @brief　利用可能なカメラIDを探索する
+   * @return 利用可能なカメラID（見つからなければ-1）
+   */
```

Comment:
```suggestion
  /**
   * @brief　利用可能なカメラIDを探索する
   * @param maxTested 探索する回数の上限
   * @return 利用可能なカメラID（見つからなければ-1）
   */
```
docstring に params が足りてないかな。こんな感じ？
他も結構足りてない。